File: aoj/aoj0003.py
N = input()
for i in range(int(N)):
    a, b, c = map(int, input().split())
    if(a*a) + (b*b) == (c*c):
        print("YES")
    elif(b*b) + (c*c) == (a*a):
        print("YES")
    elif(c*c) + (a*a) == (b*b):
        print("YES")
    else:
        print("NO")

File: aoj/php/aoj0003.php
<?php

$N = intval(trim(fgets(STDIN)));

for ($i = 0; $i < $N; $i++) {
    $input = explode(' ', trim(fgets(STDIN)));
    if (($input[0] * $input[0]) + ($input[1] * $input[1]) == $input[2] * $input[2]
        || ($input[0] * $input[0]) + ($input[2] * $input[2]) == $input[1] * $input[1]
        || ($input[1] * $input[1]) + ($input[2] * $input[2]) == $input[0] * $input[0]) {
        echo "YES" . PHP_EOL;
    } else {
        echo "NO" . PHP_EOL;
    }
}
